src/php/proxy.php

```diff
@@ -3,11 +3,16 @@
 // Get the URL that should be fetched.
 $url = isset($_GET['url']) ? $_GET['url'] : false;
 
+//create a function to evaluate the url
+function isValidURL($url){
+  return preg_match('|^https?://((www\.trumba\.com)|(scholarblogs\.emory\.edu)|(rssmix\.com))/.*$|i', $url);
```

Comment:
```suggestion
  return preg_match('#^https?://((www\.trumba\.com)|(scholarblogs\.emory\.edu)|(rssmix\.com))/.*$#i', $url);
```

Sorry, since the regex now uses pipes, we should change the delimiter to avoid confusion/escapes.